build/package.json

```diff
@@ -1,8 +1,11 @@
 {
```

Comment:
I don't think you would need to strip them out as well, they wouldn't get install by any consumer of the module